pkg/core/fs.go

```diff
@@ -28,7 +29,7 @@ const (
 	firstINode       fuseops.InodeID = 1023
 	dirDefaultMode                   = 0777 | os.ModeDir
 	fileDefaultMode                  = 0666
-	dirReadOnlyMode                  = 0755 | os.ModeDir
+	dirReadOnlyMode                  = 0777 | os.ModeDir
```

Comment:
c2ce08 is there to hopefully save some typing while preserving meaningful naming conventions on these various `*Mode` constants

Reply:
For now leaving this as two separate modes.